components/combobox/metadata/metadata.json

```diff
@@ -22,43 +22,64 @@
     ".spectrum-Combobox--sizeXL",
     ".spectrum-Combobox-button",
     ".spectrum-Combobox-button.is-focused:not(:disabled, .is-invalid, .spectrum-PickerButton--quiet)",
+    ".spectrum-Combobox-button.is-focused:not(:disabled, .is-invalid, .spectrum-PickerButton--quiet).is-hover",
```

Comment:
If you re-run using `yarn ci` are you still seeing all these additions to the metadata.json?